### ISSUE TITLE
Build failure with latest haskell-src-exts

### ISSUE THREAD

**Issue by snoyberg** [commercialhaskell/hindent#478]
```
[2 of 3] Compiling HIndent.Pretty   ( src/HIndent/Pretty.hs, dist/build/HIndent/Pretty.o )

src/HIndent/Pretty.hs:610:6: error:
    • Couldn't match expected type ‘Exp NodeInfo’
                  with actual type ‘SpecialCon l0’
    • In the pattern: ExprHole {}
      In an equation for ‘exp’: exp (ExprHole {}) = write "_"
    |
610 | exp (ExprHole {}) = write "_"
    |      ^^^^^^^^^^^

src/HIndent/Pretty.hs:865:8: error:
    • Couldn't match expected type ‘[Deriving NodeInfo]’
                  with actual type ‘Maybe a0’
    • In the pattern: Nothing
      In a case alternative: Nothing -> return ()
      In a stmt of a 'do' block:
        case mderivs of
          Nothing -> return ()
          Just derivs
            -> do newline
                  column indentSpaces (pretty derivs)
    |
865 |        Nothing -> return ()
    |        ^^^^^^^

src/HIndent/Pretty.hs:866:8: error:
    • Couldn't match expected type ‘[Deriving NodeInfo]’
                  with actual type ‘Maybe (ast0 NodeInfo)’
    • In the pattern: Just derivs
      In a case alternative:
          Just derivs
            -> do newline
                  column indentSpaces (pretty derivs)
      In a stmt of a 'do' block:
        case mderivs of
          Nothing -> return ()
          Just derivs
            -> do newline
                  column indentSpaces (pretty derivs)
```

**Comment by sophie-h** [comment]
This is supposed to be fixed in master (see #472). But there are no new releases happening (see #475).

**Comment by sighingnow** [comment]
Close as fixed.
